RFC.md

```diff
@@ -341,6 +341,183 @@ const uploadedObjectReference = await uploadTask.result;
 
 Try out the new `storage` types here: https://www.typescriptlang.org/play#gist/292f4e24178bfca5881aa20961b930dc
 
-# `API` & `Datastore` Changes
+# `API` category Changes
+
+## First param is an object with named parameters
+
+**Amplify v5 (`aws-amplify@5`)**
+
+```typescript
+const apiName = 'MyApiName';
+const path = '/path';
```

Comment:
Nit: IMO these should just be inline (e.g. `API.get('MyApiName', '/path', ...)`)

Reply:
I copy/pasted this from our docs actually. Is just to show is better the name approach so is easy to do it inline